### ISSUE TITLE
Deprecation warnings due to invalid escape sequences in python 3.8

### ISSUE THREAD

**Issue by tirkarthi** [bpython/bpython#853]
Deprecation warnings are raised due to invalid escape sequences. This can be fixed by using raw strings or escaping the literals. pyupgrade also helps in automatic conversion : https://github.com/asottile/pyupgrade/

I will raise a PR for this

```
find . -iname '*.py' | xargs -P4 -I{} python3 -Wall -m py_compile {} 
./bpython/line.py:193: DeprecationWarning: invalid escape sequence \s
  current_method_definition_name_re = LazyReCompile("def\s+([a-zA-Z_][\w]*)")
./bpython/cli.py:943: DeprecationWarning: invalid escape sequence \T
  elif key in ("KEY_NPAGE", "\T"):  # page_down or \T
./bpython/cli.py:947: DeprecationWarning: invalid escape sequence \S
  elif key in ("KEY_PPAGE", "\S"):  # page_up or \S
```

**Comment by thomasballinger** [comment]
I tried to fix this sort of thing in Curtsies and had trouble: https://github.com/bpython/curtsies/issues/128, I'll look at that again now.

**Comment by thomasballinger** [comment]
Oh I see, that's easy

**Comment by thomasballinger** [comment]
Two of these come from https://github.com/bpython/bpython/commit/0259cf810ed50dd319b88b12e96cbfd2a4e95bcc and I'm not sure what the original intent was of checking for a curses key code like `"\S"` or `"\T"`. I'm guessing they should be ^S and ^T, but ^S is usually already used for save. I'm going to remove these.

**Comment by sebastinas** [comment]
Fixed in c9a8aa3829090a9c2133cf991ffa36a51b06270a
